### ISSUE TITLE
Request: Provide Docker container for self-hosters

### ISSUE THREAD

**Issue by cryptochrome** [GilbertGobbels/GAwesomeBot#51]
It would be super efficient and easy for all self-hosters if they could simply pull a docker image and run the bot in a container. That way everything could be provided by the maintainer and self-hosters would not have to deal with stuff they have no clue about (like node.js, mongoDB etc.).

**Comment by GilbertGobbels** [comment]
While this is a good idea, and defo something to work on, simply babysitting un-experienced self-hosters is not a good idea. We're going to have to do some stuff that unexperienced self-hosters won't understand at some point.

**Comment by cryptochrome** [comment]
I agree on the babysitting part. Docker has a lot of other advantages though :)

**Comment by theanidox** [comment]
Docker would be a cool idea, as there are certain ways to add auto-updating, so people can simply wait for it to update, but I agree, we shouldn't babysit un-experienced self-hosters (I think self-hosting is easy enough to do/learn anyways).
